eval/command.go

```diff
@@ -134,6 +135,19 @@ func env(cmd string, args []value.Value) (value.Value, error) {
 	}, nil
 }
 
+// UuidCmd implements the uuid command for generating UUIDs.
+var UuidCmd = &Command{
+	Name: "uuid",
+	Argc: -1,
```

Comment:
`Argc` should be `0`, what justification would there be for allowing unlimited arguments to the `uuid` command?

Reply:
You are right of cause. This is a remainder of some previous version I did. I thought it would be good to be able to pass the UUID version one might want to create but decided against it later on and just always use UUIDv4. Will change that.